test/test.py

```diff
@@ -1,20 +1,23 @@
 import choice
 
+def c():
+  print("Delete confirmed")
+
 # Get a yes or no response (default is no)
 confirm = choice.Binary('Are you sure you want to delete?', False).ask()
 if confirm:
-    deleteIt()
+    c()
```

Comment:
No particular reason. I noticed confirming the question stops the test script as the function is undefined. A separate function is not required at all and printing a message would make sense